src/components/Navbar/Components/DrawerComponent.tsx

```diff
@@ -92,7 +94,7 @@ const DrawerComponent: FC<DrawerProps> = ({ isOpen, onClose }) => {
           </Flex>
         </Flex>
 
-        {isProjectListOpen && <ProjectCardComponent name="deneme" color="gray" />}
+        {isProjectListOpen && <ProjectCard name="deneme" color="gray" />}
```

Comment:
what is the purpose of having a name and color props?

Reply:
unnecessary props will be erased.